src/etl/ClinicalTrialModelFactory.ts

```diff
@@ -321,7 +339,11 @@ export class ClinicalTrialModelFactory {
     return ''
   }
 
-  private static emptyIfNull(fieldname: string): string {
-    return fieldname === null ? '' : fieldname
+  private static emptyIfNull(value: string): string {
+    return value === null ? '' : value
+  }
+
+  private static minusOneIfNull(value: number): number {
```

Comment:
réflexion + suggestion (non bloquante) : La notion `minusOne` me parait obscure et je ne suis pas sûr que ce soit les keywords auxquels on pense lorsqu'on cherche `-1`. Je propose de renommer par la valeur qualitative que l'on lui attribue
```suggestion
  private static emptyNumberValueIfNull(value: number): number {
```